docs/src/manual/nonlinear.md

```diff
@@ -161,6 +161,14 @@ julia> sin(sin(1.0))
 0.7456241416655579
 ```
 
+## Automatic differentiation
```

Comment:
There is a second section about AD at the bottom, I would move this down with the rest

Reply:
I kept the section. I wanted to distinguish general AD from the AD specific to operators. The bottom section was `##` instead of `###` by mistake, so it looked more prominent than I intended.